bokeh/embed.py

```diff
@@ -40,35 +40,80 @@ def components(plot_object, resources=None):
               are **already loaded**.
 
     Args:
-        plot_object (PlotObject) : Bokeh object to render
-            typically a Plot or PlotContext
+        plot_objects (PlotObject|list|dict|tuple) : Will return script
+        and div element to render if PlotObject, or correlating structures
+        of script/div combinations
```

Comment:
not sure this description is straightforward to understand.. at least not for my not english-native brain. 

I'd also suggest to explain the difference (and benefits) of passing a list/tuple or a dict and add a small example.